### PR TITLE
RFC 48: Outreach Team

### PR DESCRIPTION
**Pull request by wafflecoffee**
[Rendered view](https://github.com/wafflecoffee/rfcs/blob/patch-1/structure/0048-outreach-team.md)

This is a proposal for a new subteam of the community team. It is built on top of the process laid out for creating new teams in #47.

Initial members:
- @ToffeeMax 
- @gdude2002 
- @akarys42 & co
- @Southpaw1496
- @wafflecoffee

Unresolved questions:
- ~~How will new members be added to this team? (Should be addressed by #47)~~ Made to be a subteam of community
- Who will be the initial lead?

### CODE REVIEW

File: structure/0048-communications-team.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Explanation
+
+The Communications Team will be responsible for all communications outside of
+and within the Quilt Project. This is difficult to define without providing
+specific examples, which include:
+
+- writing and publishing blog posts
+- managing the official QuiltMC social medias, with the exception of Discord,
+  and not including moderation
```
Comment (Akarys42) [inline structure/0048-communications-team.md:30]: Does this mean Quilt moderators will have to be added to every single external media Quilt manages? In the case of, say, Reddit it could be doable although perhaps a bit hard to do, but for blocking Twitter users it isn't really possible. I think for such cases, the communication team should be capable of moderating their own spaces, assuming it is done after polling the moderation team. Another option would be to have the comm team not be allowed to moderate by themselves but be able to prompt and relay actions of the mod team when they can't action it themselves. A dedicated channel could be added on Discord for that purpose.

**Review by Akarys42 — APPROVED** [review]
Lgtm! I left a comment on moderation. I think this team will be quite necessary for the long term development of Quilt. 

We are indeed interested in being part of/leading this team.

File: structure/0048-communications-team.md (hunk 1 above)
Comment (chexo3) [reply structure/0048-communications-team.md:30]: I think communication team should be trusted to moderate their own platforms. communication team's "jurisdiction" is social media like twitter, tumblr, blog posts, etc. social media sites like Reddit or Discord with more of a community focus should be run by the community/moderation team, but could be run temporarily by communication team, with the caveat that they'd hand over control if for example, a quilt subreddit becomes an active discussion area. also, moderation/community team should be able to override the communication team on moderation decisions if necessary.

Might make sense to make communication team a sub-team of moderation/community team. Like how Catholic Cardinals aren't a "higher" rank than Bishops, they're bishops with additional responsibilities and titles.

Comment (wafflecoffee) [reply structure/0048-communications-team.md:30]: I considered making it a subteam of the community team, but then decided that it's really not in scope for that team. I mentioned the reasons for this in the document; community team is mostly moderation and events, and it shouldn't really be the responsibility of moderation and event staff to do PR. Stuff like the content of the website just... isn't a community thing.
Anyway, I do agree that communications team should have the ability to moderate when necessary, but it shouldn't be their primary responsibility. I'll work on adding that when I can.

**Review by Leo40Git — APPROVED** [review]
HR department RFC when /j
Sounds like a good idea!

File: structure/0048-communications-team.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+- writing and publishing blog posts
+- managing the official QuiltMC social medias, with the exception of Discord
+- managing relations with other platforms and communities outside of a
+  moderation context
+- writing and publishing general status updates on the progress and
+  development of Quilt
+- assisting in management of public development boards and to-do lists,
+  including GitHub project boards
+- managing the content of the website, not including the functionality
```
Comment (ToffeeMax) [inline structure/0048-communications-team.md:36]: Would this mean that the Communications Team are in charge of framing content that is currently covered by the Community Team such as Code of Conduct? (This is at least how I would read it)

**Review by ToffeeMax — CHANGES_REQUESTED** [review]
I am also seeing that you are drawing on #47 -- I presume you'll also add the nominated led to the PR after that is possibly decided.

File: structure/0048-communications-team.md (hunk 2 above)
Comment (wafflecoffee) [reply structure/0048-communications-team.md:36]: It was my impression that this would specifically be stuff like the proposed user wiki, but I suppose that if we were to make it a subteam of the Community Team (as seemed to be the consensus during our dev meeting yesterday), it could also include the code of conduct and the other things currently on the website.
CC @CheaterCodes, I feel like he would be able to answer this better than I, as this was his original idea

Comment (Kroppeb) [reply structure/0048-communications-team.md:36]: I would think that a user wiki would be moderated by the community team. I think "content" is more like a generalization of the current blog posts.

Comment (Akarys42) [reply structure/0048-communications-team.md:36]: I think it is fairly obvious that the comm team will have to make sure the content of the website matches the expectation. In the case of the CoC, it has to be decided and written by the relevant persons with the help of the community.

Maybe the wording could be changed to make it more obvious that the team is responsible for administrating the content, and only write what's relevant to them? I assume in a few cases the comm team would just take a specific messages and put them onto the website.

Comment (Kroppeb) [reply structure/0048-communications-team.md:36]: Which team do you mean with "comm team"

Comment (Akarys42) [reply structure/0048-communications-team.md:36]: ... right, communication team haha

Comment (wafflecoffee) [reply structure/0048-communications-team.md]: > but I suppose that if we were to make it a subteam of the Community Team (as seemed to be the consensus during our dev meeting yesterday

This has been done. To reduce confusion (comm team or comm team?), it's also been renamed to the Outreach Team, thanks for the idea <@902116671999860757>.

File: structure/0048-outreach-team.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+current status of development progress with the community and with any
+partners. In essence, it may be thought of as a "public relations
+department" of sorts.
+
+## Motivation
+
+This team is needed because developers would rather focus on development than
+on public relations, status updates, announcements, or team to-do lists.
+Historically, this has been managed by the Community Team as a whole.
+However, communications don't fall within the realm of the current Quilt
```
Comment (Southpaw1496) [inline structure/0048-outreach-team.md:20]: This should say "Outreach" now because the name has changed.

Comment (Leo40Git) [reply structure/0048-outreach-team.md:20]: I'm fairly sure that's communications as in communications, not as in the Communications Team.

I mean, I guess you _could_ replace it with "outreach doesn't fall within[...]", but I feel like that's a bit nitpicky.

Comment (sciwhiz12) [reply structure/0048-outreach-team.md:20]: Leaving this as "communications" would be better, since it links up with the next paragraph explaining what the Outreach Team is: "... will be responsible for all _communications_ outside of and within the Quilt project." _[emphasis mine]_

On another note regarding this paragraph, I think this last sentence needs a bit of a rewording; since the Outreach Team (which handles communications) is a subteam of the Community Team, then communications _do_ fall within the realm of the Community Team. In this case however, those responsibilities are delegated to a proper codified subteam with dedicated team members rather than managed by the Community Team's managers and moderator.

**Comment by gdude2002** [comment]
> Who will be the initial members and lead?

I'm definitely planning on being part of this team. I can act as a lead if required; the question there though really is more about who _should_ be leading such a team.

File: structure/0048-outreach-team.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+community team, and thus this should have its own separate subteam.
+
+## Explanation
+
+The Outreach Team will be responsible for all communications outside of and
+within the Quilt Project. This is difficult to define without providing
+specific examples, which include:
+
+- writing and publishing blog posts and announcements
+- managing the official QuiltMC social medias, with the exception of Discord
```
Comment (gdude2002) [inline structure/0048-outreach-team.md:30]: Could change this to "social media accounts", and drop the part about Discord - this would be better IMO since it more accurately describes things and won't need updating if we add/move to other chat platforms

File: structure/0048-outreach-team.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- managing relations with other platforms and communities outside of a
+  moderation context
+- writing and publishing general status updates on the progress and
+  development of Quilt
+- assisting in management of public development boards and to-do lists,
+  including GitHub project boards
+- managing the content of the website, not including the functionality
```
Comment (gdude2002) [inline structure/0048-outreach-team.md:37]: Why not? Who would be responsible for this if not the outreach team?

**Comment by Southpaw1496** [comment]
Well, the initial members need to be whoever is managing the systems currently covered under the RFC. So the ones I can think of:
- You, because I think you manage the Twitter account (with i5)?
- Toffee, because he writes the blog posts (although we need more people on that, I think, with just him the updates are inconsistent)
- Me, because I do the podcast.
- Someone who's familiar enough with the toolchain to manage the GitHub project boards, maybe Oro or Glitch?

File: structure/0048-outreach-team.md (hunk 4 above)
Comment (Southpaw1496) [reply structure/0048-outreach-team.md:30]: Well, the initial members need to be whoever is managing the systems currently covered under the RFC. So the ones I can think of:
- You, because I think you manage the Twitter account (with i5)?
- Toffee, because he writes the blog posts (although we need more people on that, I think, with just him the updates are inconsistent)
- Me, because I do the podcast.
- Someone who's familiar enough with the toolchain to manage the GitHub project boards, maybe Oro or Glitch?

**Comment by Southpaw1496** [comment]
> > Who will be the initial members and lead?
> 
> I'm definitely planning on being part of this team. I can act as a lead if required; the question there though really is more about who _should_ be leading such a team.

Well, the initial members need to be whoever is managing the systems currently covered under the RFC. So the ones I can think of:
- You, because I think you manage the Twitter account (with i5)?
- Toffee, because he writes the blog posts (although we need more people on that, I think, with just him the updates are inconsistent)
- Me, because I do the podcast.
- Someone who's familiar enough with the toolchain to manage the GitHub project boards, maybe Oro or Glitch?

**Comment by gdude2002** [comment]
> I think you manage the Twitter account

Twitter can actually be run in a decentralized way; i5 owns it, but Tweetdeck allows him to hand out some management perms to me (or anyone else).

I agree with you otherwise (generally speaking) - however, I think we will want someone dedicated to managing tasks/boards. It can be quite time-consuming and requires a lot of coordination with the teams you're overseeing. Heck, maybe we'd need more than one person doing that!

**Comment by Southpaw1496** [comment]
Well, you're doing a lot of things right now (Community Manager, Collab management, Cozy etc.) so maybe someone else would be more suited to the role, if it's as large as you say it is?

**Comment by gdude2002** [comment]
I wasn't putting myself forward for task/board management haha

**Comment by Southpaw1496** [comment]
Ohhhh, I see what you mean.

File: structure/0048-outreach-team.md (hunk 4 above)
Comment (chexo3) [reply structure/0048-outreach-team.md:30]: Maybe “social media, with the exception of forums and chatrooms”?

That covers Discord, and also provides the opportunity for the community team to step in with moderating, for example, the subreddit if they feel the need to.

**Comment by chexo3** [comment]
So, i5 is “keyholder” for Twitter, via TweetDeck. You want a second person (in addition to Toffee) on blog posts (maybe someone who can work with developers to make devlog posts summarizing? Essentially interviewing the members of the dev team to compile an explanation in layman’s terms of what’s going on with each project)

And no offense, but do we really need a podcast? At least at this stage. It might make sense once Quilt is in a less experimental stage, and people are starting to leverage the features that are currently being built in mods, but right now Quilt is not really having anything take advantage of those features, so there’s not much to talk about in a Quilt-focused podcast.

**Comment by gdude2002** [comment]
> do we really need a podcast?

We use them for our public dev meetings, since it's a convenient way for people to go back and listen to them later.

**Comment by Southpaw1496** [comment]
I think we may need more than two people on blog posts, Toffee hasn't been able to keep them up by himself.

**Comment by chexo3** [comment]
> > do we really need a podcast?
> 
> We use them for our public dev meetings, since it's a convenient way for people to go back and listen to them later.

Ah. Well, we could also have a separate one at some point in the future for less dev-oriented stuff. Once we have cool stuff to show off that may make sense.

File: structure/0048-outreach-team.md (hunk 5 above)
Comment (wafflecoffee) [reply structure/0048-outreach-team.md:38]: Sorry for taking ages to respond to this - this is Community Tooling, no?

**Review by Akarys42 — APPROVED** [review]
lgtm

**Comment by CheaterCodes** [comment]
Since this is a subteam of the community team, maybe the community team can propose and look for initial members/leads?
Especially if you want dedicated personal for a project board, you'll have to recruit them when this is merged latest.

The RFC itself is "good enough" I think. It would be nice to more clearly define it's boundaries, but I don't think we have enouhg experience to do that yet.

Go hire and then lgtm :)

File: structure/0048-communications-team.md (hunk 2 above)
Comment (CheaterCodes) [reply structure/0048-communications-team.md]: To answer the original question:
The content of the Code of Conduct would not be the responsibility of the outreach team.
Putting it on the website would be,
There's some details here about where it would be in it's original form (and I'd argue that should be an RFC on github).

**Review by Southpaw1496 — APPROVED** [review]

**Review by CheaterCodes — APPROVED** [review]
Looks good, lgtm

**Review by ToffeeMax — APPROVED** [review]
I have reviewed the changes and give approval for this RFC be put to vote.

**Comment by OroArmor** [comment]
This RFC is now in FCP

**Review by Kroppeb — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Review by gdude2002 — APPROVED** [review]

**Comment by Southpaw1496** [comment]
Did we ever finalise who was going to lead the team? I know Gdude offered but he has a lot to do already. 

I would be quite interested in doing it, if no-one objected.

**Comment by gdude2002** [comment]
Well, you've had a couple of thumbs-up so far!

**Comment by Southpaw1496** [comment]
And no thumbs-down :D
